corehq/apps/fixtures/download.py

```diff
@@ -277,7 +269,6 @@ def empty_padding_list(length):
             + group_headers
             + location_headers
         )
-        excel_sheets[data_type.tag] = item_sheet
```

Comment:
It's not necessary because it sets the two equal again at the end of each loop and it doesn't change anything about `item_sheet['headers']` after setting it above so it should be safe.